code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -155,7 +155,7 @@
 						return held_weapon.afterattack(target,src)
 
 			var/disarm_chance = rand(1, 100)
-			var/attacker_skill_level = attacking_mob.skills ? skills.get_skill_level(SKILL_CQC) : SKILL_CQC_MAX // No skills, so assume max
+			var/attacker_skill_level = skills && attacking_mob.skills ? skills.get_skill_level(SKILL_CQC) : SKILL_CQC_MAX // No skills, so assume max
```

Comment:
What was this change for?

Reply:
When playing as CO, and trying to disarm the dummy, skills was null. Im assuming skills is referring to the dummy, which would make sense. Either way seems like a check was needed for it as it was erroring.